src/SmearedJetProducer.cc

```diff
@@ -205,7 +207,7 @@ TLorentzVector SmearedJetProducer::getSmearedJetFromIdx(unsigned int jetIdx, big
   TLorentzVector jet(theBigTree.jets_px->at(jetIdx), theBigTree.jets_py->at(jetIdx), theBigTree.jets_pz->at(jetIdx), theBigTree.jets_e ->at(jetIdx));
 
   // Get the smear factor
-  double smearFactor = SmearedJetProducer::getSmearFactor(jet, theBigTree);
+  double smearFactor = SmearedJetProducer::getSmearFactor(jet, theBigTree, true);
```

Comment:
are we using `getSmearedJetFromIdx` anywhere? and if so is it ok to always set this to `true`?